content/rs/release-notes/rs-7-4-2-releases/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+Title: Redis Enterprise Software release notes 7.4.2
+linkTitle: 7.4.2 releases
+description: New Cluster Manager UI enhancements, including Active-Active database management. TLS 1.3 support. Automatic recovery configuration. IPv6 support for internal traffic. Maintenance mode enhancements. Two feature sets.
+compatibleOSSVersion: Redis 7.2.0
+weight: 70
+alwaysopen: false
+toc: "true"
+categories: ["RS"]
+---
+
+​[​Redis Enterprise Software version 7.4.2](https://redis.com/redis-enterprise-software/download-center/software/) is now available!
+
+## Highlights
+
+This version offers:
+ 
+- New Cluster Manager UI enhancements, including Active-Active database management
+
+- TLS 1.3 support
+
+- Automatic recovery configuration
+
+- IPv6 support for internal traffic
+
+- Maintenance mode enhancements
+
+- Two feature sets
```

Comment:
@rrelledge I don't think customers will get this title, maybe 'module management enhancement' or bundling Akita and Beagle feature set?

Reply:
Fixed